dex/elrond_dex_pair/src/lib.rs

```diff
@@ -733,11 +675,6 @@ pub trait Pair:
         self.state().get() == State::Active
     }
 
-    #[view]
```

Comment:
there is no need to view the lp token id?

Reply:
moved the function. it's still used